frameworks/g-cloud-7/manifests/edit_service_as_admin.yml

```diff
@@ -35,3 +35,12 @@
     - termsAndConditionsDocumentURL
     - sfiaRateDocumentURL
     - pricingDocumentURL
+-
+  name: Support
+  editable: True
```

Comment:
Should this be lowercase? 
It doesn't seem to matter to our apps, but we've always written it as lowercase everywhere else.